hits_ingest.py

```diff
@@ -5,88 +5,102 @@
 from lsst.pipe.tasks.ingest import IngestConfig
 from lsst.pipe.tasks.ingestCalibs import IngestCalibsConfig, IngestCalibsTask
 from lsst.pipe.tasks.ingestCalibs import IngestCalibsArgumentParser
-from lsst.pipe.tasks.processCcd import ProcessCcdTask
-from lsst.pipe.base import ArgumentParser
+from lsst.pipe.tasks.processCcd import ProcessCcdTask, ProcessCcdConfig
+from lsst.meas.algorithms import LoadIndexedReferenceObjectsTask
 from lsst.utils import getPackageDir
+from lsst.ip.diffim.getTemplate import GetCalexpAsTemplateTask
+from lsst.pipe.tasks.imageDifference import ImageDifferenceConfig, ImageDifferenceTask
 from glob import glob
-import sys
 import sqlite3
 import os
+import argparse
 '''
```

Comment:
There are all these nice strings describing how to use the script, but right now the user has to open up the code to read them. If one could just call `python hits_ingest.py --help` and see some nice text, that would be helpful. Argparse supports this.